### DISCUSSION TITLE
Problem/error updating AdGuard Home to v0.107.46

### DISCUSSION THREAD

**Discussion by leomeneguzzi** [AdguardTeam/AdGuardHome#6842] [Q&A]
Hello, when I click on the "Update now" button, it just gives me a 500 HTTP error with this message: `getting path: no such file or directory`

![Screenshot 2024-03-21 at 12 13 50](https://github.com/AdguardTeam/AdGuardHome/assets/7086176/4d0fda2d-90bb-497c-8133-cf5e65633634)

Someone also had this problem?

**Comment by ainar-g** [comment]
Do you happen to run it on macOS?  If so, make sure that your installation is within the `/Application` directory, [as documented](https://adguard-dns.io/kb/adguard-home/getting-started/#official-releases).

**Reply by leomeneguzzi** [reply]
@ainar-g I'm running with the OPNSense, so it's on FreeBSD.
Sorry, I forgot to mention it.

**Comment by leomeneguzzi** [comment]
Just now, I clicked on "Update" again and it worked
😕
